website/public/js/controllers/tasksCtrl.js

```diff
@@ -129,6 +132,24 @@ habitrpg.controller("TasksCtrl", ['$scope', '$rootScope', '$location', 'User','N
 
     /*
      ------------------------
+     Dailies
+     ------------------------
+     */
+    $scope.updateTaskStartDate = function(task) {
+      // TODO: Weirdness here...startDate should be a Date and _dateString
+      // should be a string, but the datePicker input fields sets _dateString
+      // to a Date even though input type="text"...
+      task.startDate = task._dateString;
+    };
+
+    $scope.open = function($event) {
```

Comment:
1. Sure. Good catch, open() is a pretty terrible name.
2. Ooh, got it! I'm a beginner at angular, so still making newbie mistakes.